src/badges.js

```diff
@@ -0,0 +1,953 @@
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import path from 'path';
+import fs, { promises as fsp } from 'fs';
+import { exit } from 'process';
+import { wait, copy, copyTransform } from './utils/index.js';
+import { ConfigApi } from './services/ConfigApi.js';
+import { buildLibrary } from './library.js';
+
+const TEMPLATE_TYPE_BADGES = 'snap/badge';
+const DIR_EXCLUDE_LIST = ['node_modules', '.git'];
+
+const DEFAULT_LOCATIONS = {
+	overlay: {
+		left: [
+			{
+				name: 'left',
+				label: 'left',
+			},
+		],
+		right: [
+			{
+				name: 'right',
+				label: 'right',
+			},
+		],
+	},
+	callout: [
+		{
+			name: 'callout',
+			label: 'callout',
+		},
+	],
+};
```

Comment:
I'm thinking this shouldn't be in here at all, and we should always just fetch the location out of smc-config-api.

Reply:
For example, the default may change, and different sites may end up with different defaults.